source/hands_on/td3_bc.rst

```diff
@@ -0,0 +1,104 @@
+TD3BC
+^^^^^^^
+
+Overview
+---------
+
+TD3BC, proposed in the 2021 paper `A Minimalist Approach to Offline Reinforcement Learning <https://arxiv.org/abs/2106.06860>`_,
+is a simple approach to offline RL where only two changes are made to TD3: a weighted behavior cloning loss is added to the policy update and the states are normalized.
+Unlike competing methods there are no changes to architecture or underlying hyperparameters.
+The resulting algorithm is a simple baseline that is easy to implement and tune, while more than halving the overall run time by removing the additional computational overhead of previous methods.
+
+Quick Facts
+-----------
+1. TD3BC is an **offline** RL algorithm.
+
+2. TD3BC is based on **TD3** and **behavior cloning**.
+
+Key Equations or Key Graphs
+---------------------------
+TD3BC simply consists to add a behavior cloning term to TD3 in order to regularize the policy:
+
+.. math::
+    \begin{aligned}
+    \pi = \arg\max_{\pi} \mathbb{E}_{(s, a) \sim D} [ \lambda Q(s, \pi(s)) - (\pi(s)-a)^2 ]
+    \end{aligned}
+
+Additionally, all the states in the dataset are normalized, such that they have mean 0 and standard deviation 1.
```

Comment:
state normalization is applied in each mini-batch, current statements may lead to misunderstanding, such as norm in the whole dataset.